### ISSUE TITLE
About the loss function of LSGAN

### ISSUE THREAD

**Issue by zhangqianhui** [github-pengge/PyTorch-progressive_growing_of_gans#6]
You code about the D_loss

`d_adv_loss = self.compute_adv_loss(self.d_real, True, 0.5) + self.compute_adv_loss(self.d_fake, False, 0.5)`

The official code of  D_loss

```
 if type == 'lsgan': 
        G_loss = L2(fake_scores_out, 0)
        D_loss = L2(real_scores_out, 0) + L2(fake_scores_out, 1) * L2_fake_weight
```
The value of L2_fake__weight is 0.1.

So, Do you abserve it?

**Comment by github-pengge** [comment]
I didn't notice that. Thanks.
